.github/workflows/main.yml

```diff
@@ -22,20 +22,29 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest]
-        python_version: ['3.8', '3.9', '3.10', '3.11']
+        python_version: ['3.9', '3.10', '3.11']
```

Comment:
We can probably take the opportunity to test on newer Python versions like 3.13 (but can be done in a separate PR).